examples/imagenet/train_imagenet.py

```diff
@@ -24,6 +25,191 @@
 import resnet50
 import resnext50
 
+try:
```

Comment:
I feel that adding to much.
- Create new train script as `train_imagenet_dali.py` .
- Move these code to separate utility file `dali_util.py` .